easybuild/easyconfigs/m/MXNet/MXNet-0.9.3-foss-2016b-Python-2.7.12-R-3.3.3.eb

```diff
@@ -21,11 +21,11 @@ sources = [
 ]
 
 checksums = [
-    "84971e4a0e2e9db3757623657aa5b9ec",  # v0.9.3.tar.gz
-    "12c96664978c545a7269b93635719b18",  # 6856021ca925c7.tar.gz
-    "3dc48183e123c650707bcf26a9254941",  # c29ac58320d7b0.tar.gz
-    "fb04cb1694f1e66591a31a18bc92b451",  # 0d64855f741e04.tar.gz
-    "da873c881758c0e10ac0928f86568456",  # b32ec4aee65a66.tar.gz
+    'c8d1e6d8a178e475625e60a8fd71b49efbd7c8a2b222300858f50201d89127e3',  # v0.9.3.tar.gz
```

Comment:
apparently this new tarball is different than the one that was previously available, i.e. it now contains a directory named `incubator-mxnet-0.9.3` whereas before it contained a directory named `mxnet-0.9.3`...

Because of this, the MXNet easyblock needs to be updated, see https://github.com/easybuilders/easybuild-easyblocks/pull/1257